src/Provider/Provider.Web/Orchestrators/VacancyManageOrchestrator.cs

```diff
@@ -52,9 +61,24 @@ public async Task<ManageVacancyViewModel> GetManageVacancyViewModel(Vacancy vaca
             viewModel.CanShowCloseVacancyLink = vacancy.CanClose;
             viewModel.CanShowCloneVacancyLink = vacancy.CanClone;
 
-            var vacancyApplicationsTask =  await _client.GetVacancyApplicationsAsync(vacancy.VacancyReference.Value.ToString());
+            var applications = new List<VacancyApplication>();
 
-            var applications = vacancyApplicationsTask?.Applications ?? new List<VacancyApplication>();
+            if (_featureToggle.IsFeatureEnabled(RecruitFeatureNames.CanShowVacancyAnalytics) && vacancy.ApprovedDate > _systemConfig.ShowAnalyticsForVacanciesApprovedAfterDate)
+            {
+                var vacancyApplicationsTask = _client.GetVacancyApplicationsAsync(vacancy.VacancyReference.Value.ToString());
```

Comment:
As mentioned above, the tasks can be collected in a list and awaited instead of repetition of vacancyApplication task